lib/ae_mdw/db/name.ex

```diff
@@ -325,17 +317,10 @@ defmodule AeMdw.Db.Name do
     end
   end
 
-  @spec revoke_or_expire_height(nil | {Blocks.block_index(), AeMdw.Txs.txi()}, Blocks.height()) ::
-          Blocks.height()
-  def revoke_or_expire_height(nil = _revoke, expire),
-    do: expire
-
-  def revoke_or_expire_height({{height, _}, _}, _expire),
-    do: height
-
   @spec revoke_or_expire_height(Model.name()) :: Blocks.height()
-  def revoke_or_expire_height(m_name),
-    do: revoke_or_expire_height(Model.name(m_name, :revoke), Model.name(m_name, :expire))
+  def revoke_or_expire_height(m_name) do
+    do_revoke_or_expire_height(Model.name(m_name, :revoke), Model.name(m_name, :expire))
```

Comment:
Why do we use the `do_` prefix here? Perhaps we can pattern match this function and avoid using non-reusable auxiliary functions?

Reply:
when they were together it helped me reading which was the caller and the called but now with one of them private I agree it's not needed anymore. thanks!